ad/movement/trusts.md

```diff
@@ -198,6 +207,50 @@ When doing NTLM authentications across trusts, the trusting domain's domain cont
+
+## Trust Attack High-Level Strategy (adapted by [Will Schroeder's " A Guide to Attacking Domain Trusts"](https://harmj0y.medium.com/a-guide-to-attacking-domain-trusts-ef5f8992bb9d))
+1. The first step is to enumerate all trusts the current domain has, along with any trusts those domains have, and so on. Essentially, this includes the process of producing a mapping of all the domains that are reachable from the current context through the linking of trust referrals. This allows one to determine the domains they need to hop through to get to their target and what techniques they can execute to achieve this. Any domains in the mapped “mesh” that are in the same forest (e.g. parent->child relationships) are of particular interest due to the `SIDhistory-trust-hopping` technique.
+2. The next step is to enumerate any users/groups/computers (security principals) in one domain that either (1) have access to resources in another domain (i.e. membership in local administrator groups, or DACL ACE entries), or (2) are in groups or (if a group) have users from another domain. The point here is to find relationships that cross the mapped trust boundaries in some way and therefore might provide a type of “access bridge” from one domain to another in the mesh. While a cross-domain nested relationship is not guaranteed to facilitate access, trusts are normally implemented for a reason, meaning more often than not some type of cross-domain user/group/resource “nesting” probably exists, and in many organizations these relationships are misconfigured. Kerberoasting across trusts may be another vector to hop a trust boundary.
+3. When the trust mesh, types, and cross-domain nested relationships have been mapped out, one can have a clear map of what accounts need to be compromised to pivot from our current domain into our target.
+4. At a minimum, remember that if a domain trusts us, i.e. if the trust is bidirectional or if one-way and inbound, then one can query any Active Directory information from the trusting domain. ALL parent->child (intra-forest domain trusts) retain an implicit two-way transitive trust with each other. Furthermore, due to how child domains are added, the “Enterprise Admins” group is automatically added to Administrators domain local group in each domain in the forest. This means that trust “flows down” from the forest root, making it a primary objective to move from child to forest root at any appropriate step in the attack chain.
```

Comment:
This needs to be inserted at the beginning of the Practice section in this page instead of having it as a subsection of the Bastion Forest.

Reply:
This one is a general Trusts attack methodology and does not apply to Bastion Forests. Should I put it in the practice section under the same title? I am thinking of placing it above the enumeration section since it is a general overview of the methodology an auditor should follow.